### PR TITLE
feat(apps): app editor improvements

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

Small improvements:
- export `Link` tag from `@posthog/apps-common`
- can import from `kea-*` packages
- will wait up to 2min for reloads instead of the ridiculous 10sec (we should find ways to improve this via async compilation jobs that don't depend on full plugin reloads)
- copied prettier properties from `.prettierrc`. notably `printWidth: 120`
- typescript support for React
image

## How did you test this code?

TBD

### CODE REVIEW

**Review by pauldambra — APPROVED** [review]
couple of comments thinking out loud

but nothing to block this 👍

File: frontend/src/scenes/plugins/source/types/packages.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
[the remark this answers is not in the document]
Comment (mariusandra) [reply frontend/src/scenes/plugins/source/types/packages.json:1]: I'm planning to make a script to autogenerate this file, and will add better instructions then. Got stuck when trying to build the types for `@posthog/apps-common`...

File: frontend/src/scenes/apps/frontendAppRequire.ts [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+    kea: allKea,
+    'kea-forms': allKeaForms,
+    'kea-loaders': allKeaLoaders,
+    'kea-router': allKeaRouter,
+    'kea-subscriptions': allKeaSubscriptions,
+    'kea-window-values': allKeaWindowValues,
+    '@posthog/apps-common': appsCommon,
+}
+
 /** Every `import` in a frontend app will be piped through here */
```
[the remark this answers is not in the document]
Comment (mariusandra) [reply frontend/src/scenes/apps/frontendAppRequire.ts:21]: Definitely something to consider as we un-beta this.

File: frontend/src/scenes/plugins/source/types/packages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariusandra) [reply frontend/src/scenes/plugins/source/types/packages.json:1]: Seems like I'm not sure what I did anymore :D. I did merge in the change to export types automatically 😅 
Will add a README in one of the next PRs.

**Pull request merged**